docs/prerequisites.md

```diff
@@ -1,93 +1,88 @@
 
-2. Create AWS Load Balancer Operator's namespace:
-
-    ```bash
-    oc create namespace aws-load-balancer-operator
-    ```
-
-3. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects needed to bootstrap the operator:
+2. Use the `ccoctl` tool to create a IAM role from the operator's `CredentialsRequest`:
 
     ```bash
-    curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
-    ccoctl aws create-iam-roles \
+   $ curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+   $ ccoctl aws create-iam-roles \
         --name <name> --region=<aws_region> \
         --credentials-requests-dir=<path-to-credrequests-dir> \
         --identity-provider-arn <oidc-arn>
+
+    2023/09/12 11:38:57 Role arn:aws:iam::777777777777:role/<name>-aws-load-balancer-operator-aws-load-balancer-operator created
+    2023/09/12 11:38:57 Saved credentials configuration to: /home/user/credrequests-dir/manifests/aws-load-balancer-operator-aws-load-balancer-operator-credentials.yaml
+    2023/09/12 11:38:58 Updated Role policy for Role <name>-aws-load-balancer-operator-aws-load-balancer-operator created
     ```
 
     For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust
     policy that is tied to the specified OIDC identity provider, and permissions
     policy as defined in each `CredentialsRequest` object. This also generates a set
-    of secrets in a **manifests** directory that is required
-    by the **aws-load-balancer-operator**.
+    of secrets in a `manifests` directory, which are not needed by the operator.
 
-4. Apply the secrets to your cluster:
+    **Note**: Write down the role ARN for the aws-load-balancer-operator in the output, it will be needed for the next step.
 
-    ```bash
-    ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
-    ```
-
-5. Verify that the operator's credentials secret is created:
+3. Install the operator using the OpenShift OperatorHub web UI or by running the following commands:
 
     ```bash
-    oc -n aws-load-balancer-operator get secret aws-load-balancer-operator -o json | jq -r '.data.credentials' | base64 -d
-    [default]
-    sts_regional_endpoints = regional
-    role_arn = arn:aws:iam::999999999999:role/aws-load-balancer-operator-aws-load-balancer-operator
-    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+    $ oc create namespace aws-load-balancer-operator
+
+    $ cat <<EOF | oc apply -f -
+    apiVersion: operators.coreos.com/v1
+    kind: OperatorGroup
+    metadata:
+      name: aws-load-balancer-operator
+      namespace: aws-load-balancer-operator
+    spec:
+      targetNamespaces: []
+    EOF
+
+    $ ROLEARN="<iam-role-arn>"
```

Comment:
Can we provide more info or detailed commands for getting the `ROLEARN` ? Is this only required for STS cluster ?

Reply:
Added a step above to set `ROLEARN` environment variable.